src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/TrafficSeo/Meta/MetaSettingsShopUrlsFormDataProvider.php

```diff
@@ -121,6 +121,14 @@ private function validateData(array $data)
             );
         }
 
+        if (!$this->validate->isURL($data['physical_uri'])) {
+            $errors[] = $this->translator->trans(
+                'The Base URI is not valid.',
+                [],
+                'Admin.Notifications.Error'
```

Comment:
Shouldn't it be localized in Admin.Shopparameters.Notification? What do you think @LouiseBonnard ?

Reply:
It can be both. Usually, we **use general _Admin.Notifications.Error_ domain when the error notification is commonly used** throughout the software. If the **notification is very specific to a page, we'd better pick the proper subdomain**, here _Admin.Shopparameters.Notification_.

So I think the same, this string should be localized in _Admin.Shopparameters.Notification_. :-)

Reply:
Thank you!